local-cli/runWindows/utils/build.js

```diff
@@ -1,39 +0,0 @@
-'use strict';
-
-const execSync = require('child_process').execSync;
-const path = require('path');
-const chalk = require('chalk');
-const glob = require('glob');
-const MSBuildTools = require('./msbuildtools');
-const Version = require('./version');
-
-function buildSolution(slnFile, buildType, buildArch, verbose) {
-  const minVersion = new Version(10, 0, 10586, 0);
-  const allVersions = MSBuildTools.getAllAvailableUAPVersions();
-  if (!allVersions.some(v => v.gte(minVersion))) {
-    throw new Error('Must have a minimum Windows SDK version 10.0.10586.0 installed');
-  }
-
-  console.log(chalk.green(`Building ${slnFile}`));
-  const msBuildTools = MSBuildTools.findAvailableVersion();
-  msBuildTools.buildProject(slnFile, buildType, buildArch, null, verbose);
```

Comment:
`build` never passed in config to `msbuildtools.buildProject(slnFile, buildType, buildArch, config, verbose)`, so the new version doesn't even consider the configs